lib/utils/yamlAstParser.js

```diff
@@ -90,7 +90,7 @@ const addNewArrayItem = (ymlFile, pathInYml, newValue) =>
     for (let i = 0; i < pathInYmlArray.length - 1; i++) {
       const propertyName = pathInYmlArray[i];
       const property = currentNode[propertyName];
-      if (_.isUndefined(property) || _.isObject(property)) {
+      if (property === undefined || _.isObject(property)) {
```

Comment:
Here we can remove _undefined_ check from condition (if value is an _object_ it certainly is not `undefined`)

Reply:
As I understand in this case, we allow `property` whose value is `undefined` or an *object* pass right? So we can't remove the undefined check here.

Reply:
Indeed, you're right. Sorry for confusion